scripts/js/commands/checkPagesRender.ts

```diff
@@ -39,7 +39,9 @@ const readArgs = (): Arguments => {
       type: "string",
       normalize: true,
       description:
-        "Read the file path for file paths and globs to check, like `docs/start/index.md`. Entries should be separated by a newline and should be valid file types (.md, .mdx, .ipynb).",
+        "Read the file path for file paths and globs to check, like `docs/start/index.md`. " +
+        "Entries should be separated by a newline and should be valid file types (.md, .mdx, .ipynb)." +
+        "Mutually exclusive with the other args.",
```

Comment:
Nice comment! Maybe in the future, we can refactor it and use `.conflcts()` to error if `--from-file` is used with another argument to avoid setting them unintentionally.

https://yargs.js.org/docs/#api-reference-conflictsx-y

Reply:
I think, in https://github.com/Qiskit/documentation/pull/2389/commits/5b577010386927e99b50c97fee46f4c2a5f288a0,  we also need to remove the `default: false` from the other arguments and let them be `undefined`. Otherwise, they count as set by `conflicts` and the option `from-files` will always error